UIKItTemplate/Modules/NewsLine/View/Cell/PostViewCell.swift

```diff
@@ -0,0 +1,291 @@
+
+    // MARK: - Private Properties
+
+    private var post: Post? {
+        didSet {
+            if let post {
+                setupUI(post: post)
+            }
+        }
+    }
+
+    // MARK: - Public methods
+
+    func setupWith(_ post: Post?) {
+        self.post = post
+    }
+
+    // MARK: - Private Methods
+
+    private func setupUI(post: Post) {
+        contentView.addSubviews(
+            avatarImageView,
+            nickNameLabel,
+            moreButton,
+            imagesScrollView,
+            likeButton,
+            messageButton,
+            sendButton,
+            bookmarkButton,
+            pageControl,
+            likesLabel,
+            descriptionLabel,
+            smallAvatarImageView,
+            commentLabel,
+            timeLabel
+        )
+        setupUIData(post: post)
+        contentView.disableTARMIC()
+
+        setupConstraints()
+    }
+
+    private func setupUIData(post: Post) {
+        avatarImageView.image = UIImage(named: post.avatarImageName)
+        nickNameLabel.text = post.nickName
+        makeSubviews(post: post)
+        pageControl.numberOfPages = post.imageNames.count
+        descriptionLabel.attributedText = makeDescriptionLabelText(post: post)
+        timeLabel.text = post.timeStamp
+    }
+
+    private func makeSubviews(post: Post) {
+        let screenWidth = UIScreen.main.bounds.width
+        for (index, image) in post.imageNames.enumerated() {
+            let imageView = UIImageView()
+            imageView.image = UIImage(named: image)
+            imageView.translatesAutoresizingMaskIntoConstraints = false
+            imagesScrollView.addSubview(imageView)
+
+            // setup constraints
+            let leadingInset = CGFloat(index) * screenWidth
+            imageView.leadingAnchor.constraint(equalTo: imagesScrollView.leadingAnchor, constant: leadingInset)
+                .isActive = true
+            imageView.topAnchor.constraint(equalTo: imagesScrollView.topAnchor).isActive = true
+            imageView.bottomAnchor.constraint(equalTo: imagesScrollView.bottomAnchor).isActive = true
+            imageView.widthAnchor.constraint(equalToConstant: screenWidth).isActive = true
+            imageView.heightAnchor.constraint(equalTo: imageView.widthAnchor, multiplier: Constants.postImageRation)
+                .isActive = true
+            if index == post.imageNames.count - 1 {
+                imageView.trailingAnchor.constraint(greaterThanOrEqualTo: imagesScrollView.trailingAnchor)
+                    .isActive = true
+            }
+        }
+    }
+
+    private func makeDescriptionLabelText(post: Post) -> NSMutableAttributedString {
+        let boldFont = UIFont.makeVerdanaBold(10)
+        let boldAttributes = [NSAttributedString.Key.font: boldFont]
+        let output = NSMutableAttributedString(
+            string: post.nickName,
+            attributes: boldAttributes as [NSAttributedString.Key: Any]
```

Comment:
у тебя в данном случае атрибуты короткие и не переиспользуются
но из-за того, что ты их описал в отдельной переменной, приходится делать as и дважды писть полый тип. Поэтому в конкретно этой ситуации, мне кажется, что было бы более читабельно и емко атрибуты сразу в инициализатор передать литералом
```suggestion
            attributes: [.font: boldFont]
```

Reply:
О, крутая тема поправил везде